lib/neopixel/blockingneopixel.cpp

```diff
@@ -0,0 +1,170 @@
+#include "blockingneopixel.h"
+
+#define H_1_NS 400
+#define L_1_NS 100
+#define H_0_NS 150
+#define L_0_NS 300
+
+DigitalOut out(PC_8);
+
+BlockingNeopixel::BlockingNeopixel(int numPixels)
+{
+    pixelNum = numPixels;
+}
+
+BlockingNeopixel::~BlockingNeopixel()

```

Comment:
why the empty destructor?

Reply:
I'm not sure what to put in the destructor and I think the default destructor will still clean up the object itself.

Reply:
It's not needed since you didn't create anything dynamically. You can leave like this or remove it and it will be the same.